core/lib/Thelia/Model/FeatureProductQuery.php

```diff
@@ -16,5 +18,60 @@
  */
 class FeatureProductQuery extends BaseFeatureProductQuery
 {
+    /**
+     * @inheritdoc
+     * @deprecated since version 2.4.0, to be removed in 3.0.
+     *                      Please use  filterByIsFreeText() instead
+     */
+    public function filterByFreeTextValue($freeTextValue = null, $comparison = null)
+    {
+        $bt = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 1);
+        Tlog::getInstance()->warning(sprintf('Using `free_text_value` is deprecated. Use `is_free_text` instead. Used in %s:%d', $bt[0]['file'], $bt[0]['line']));
+        return parent::filterByFreeTextValue($freeTextValue, $comparison);
+    }
+
+    /**
+     * @inheritdoc
+     * @deprecated since version 2.4.0, to be removed in 3.0.
+     *                      Please use  orderByIsFreeText() instead
+     */
+    public function orderByFreeTextValue($order = Criteria::ASC)
+    {
+        $bt = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 1);
+        Tlog::getInstance()->warning(sprintf('Using `free_text_value` is deprecated. Use `is_free_text` instead. Used in %s:%d', $bt[0]['file'], $bt[0]['line']));
+        return parent::orderByFreeTextValue($order);
+    }
+
+    /**
+     * @inheritdoc
+     * @deprecated since version 2.4.0, to be removed in 3.0.
+     *                      Please use  groupByIsFreeText() instead
+     */
+    public function groupByFreeTextValue()
+    {
+        $bt = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 1);
+        Tlog::getInstance()->warning(sprintf('Using `free_text_value` is deprecated. Use `is_free_text` instead. Used in %s:%d', $bt[0]['file'], $bt[0]['line']));
+        return parent::groupByFreeTextValue();
+    }
+
+    /**
+     * @inheritdoc
+     * @deprecated since version 2.4.0, to be removed in 3.0.
+     *                      Please use  findByIsFreeText() instead
+     */
+    public function findByFreeTextValue($free_text_value)
+    {
+        return parent::findByFreeTextValue($free_text_value);
```

Comment:
Deprecation already logged in filterByFreeTextValue. But it show wrong place of caller (propel internals)... Ok, I'll add additional logs for that methods.